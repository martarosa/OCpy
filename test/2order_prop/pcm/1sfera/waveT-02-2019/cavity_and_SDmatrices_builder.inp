&MEDIUM
medium_type='sol'
medium_pol='chr'
bem_read_write='wri'
/

&SURFACE
input_surface='bui'
spheres_number=1
sphere_position_x=0.
sphere_position_y=0.
sphere_position_z=0.
sphere_radius=10
/

&EPS_FUNCTION
/
