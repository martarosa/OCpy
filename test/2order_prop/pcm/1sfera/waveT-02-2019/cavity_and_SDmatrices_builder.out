 Solvent as external medium
 Medium polarization described by apparent charges
 Diagonal BEM formulation
 This is a Classical BEM run
 This run just writes matrices and boundary
 Building surface from spheres.
 Number of spheres           1
 I_sphere  X     Y   Z   Radius
     1      0.0000      0.0000      0.0000     10.0000
 Essential output written.
Done reading input, took     0.000s
 GEPOL-GB: COUNTING TESSERAE
 nts_act          60
 GEPOL-GB: GENERATING TESSERAE

 TOTAL NUMBER OF SPHERES=    1
 SPHERE             CENTER  (X,Y,Z) (A)               RADIUS (A)      AREA(A*A)
   1    0.000000000    0.000000000    0.000000000    5.291772492

 TOTAL NUMBER OF TESSERAE=      60
 SURFACE AREA=        351.89426816(A**2)    CAVITY VOLUME=        620.71480282 (A**3)
 nts_act          60
 Created output file with surface points
 Matrixes S D have been written out
